modules/mod_acl_user_groups/mod_acl_user_groups.erl

```diff
@@ -601,19 +601,22 @@ manage_datamodel(Context) ->
 
             predicates=
                 [
-                    {hasusergroup, 
+                    {hasusergroup,
                         [{title, {trans, [{en, <<"In User Group">>},{nl, <<"In gebruikersgroep">>}]}}],
                         [{person, acl_user_group}]
                     },
-                    {hascollabmember, 
+                    {hascollabmember,
                         [{title, {trans, [{en, <<"Member">>},{nl, <<"Lid">>}]}}],
                         [{acl_collaboration_group, person}]
                     },
-                    {hascollabmanager, 
+                    {hascollabmanager,
                         [{title, {trans, [{en, <<"Manager">>},{nl, <<"Beheerder">>}]}}],
                         [{acl_collaboration_group, person}]
                     }
-                ]
+                ],
+            data = [
+                {acl_rules, acl_default_rules:get_default_rules()}
+            ]
```

Comment:
Aren't rules that are inserted programmatically causing that the rules cannot be edited in the UI anymore?